CodeEdit.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -3,13 +3,22 @@
     "pins": [
       {
         "package": "Highlightr",
-        "repositoryURL": "https://github.com/raspu/Highlightr.git",
+        "repositoryURL": "https://github.com/raspu/Highlightr",
```

Comment:
leave `.git`

Reply:
<img width="481" alt="CleanShot 2022-03-29 at 20 13 27@2x" src="https://user-images.githubusercontent.com/46642225/160608703-18fa0d16-3e90-4293-a96b-f4cd293a98da.png">

It seems work well in my mac.